### ISSUE TITLE
What to do with measureit?

### ISSUE THREAD

**Issue by lalelunet** [lalelunet/measureit#28]
As announced on the project page feel free to suggest here the future of measureit

**Comment by Warren735** [comment]
Thomas
Is Measurit being supported?
I have a project in mind using a Rasberry Pi 3 and CurrentCost monitor or some alternative power measurement system.

**Comment by lalelunet** [comment]
Yes. I support measureit. There is also a a forum where a lot of questions are already answered :)
https://groups.google.com/forum/#!forum/measureit

**Comment by Warren735** [comment]
Thanks Thomas.
Unfortunately I know of no alternatives in the price bracket.  There are plenty of  options if you want to spend $1500 or so.  What I really want is a bidirectional meter.  I am hoping that with some tricks I may be able to use your software and some historical data to determine the direction of current flow.   
Apologies for drifting off topic.  I will follow it up on the Google groups form.
